### PR TITLE
Documentation updating, Hardware management for OpenPower systems

### PR DESCRIPTION
**Pull request by neo954**
The only file changed in this updating is

    docs/source/guides/admin-guides/manage_clusters/ppc64le/management.rst

### CODE REVIEW

**Comment by neo954** [comment]
If anyone of the reviewer wants to read a rendered HTML page, please check the [attachment](https://github.com/xcat2/xcat-core/files/13028/management.html.txt).

File: docs/source/guides/admin-guides/manage_clusters/ppc64le/management.rst [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+````````````````
+
+Most of modern enterprise level server machine has an LED installed on its front panel and/or rear panel. Which is called beacon light. While this light has been turned on, the system adminstrator can use this light to indicate one physcial machine out of a bunch of enclosures in a server frame. And it made life easier.
+
+With xCAT, the end user can turn the beacon light on or off with the commands show below. ::
+
+    rbeacon cn1 on
+    rbeacon cn1 off
+
+Please notice, the current state of the beacon light can not be inquery remotely. As a workaround, one can always use the `rbeacon` command to trun all the beacon lights in one frame off, and then turn a particular beancon light on. ::
```
Comment (whowutwut) [inline docs/source/guides/admin-guides/manage_clusters/ppc64le/management.rst]: should we make the commands wrapped in two tick marks to show code format.  You have a single back tick there. ``rbeacon``

File: docs/source/guides/admin-guides/manage_clusters/ppc64le/management.rst [hunk 2]
```diff
@@ -0,0 +1,113 @@
+Hardware Management
+===================
+
+Basic Operation
+---------------
+
+The Beacon Light
+````````````````
+
+Most of modern enterprise level server machine has an LED installed on its front panel and/or rear panel. Which is called beacon light. While this light has been turned on, the system adminstrator can use this light to indicate one physcial machine out of a bunch of enclosures in a server frame. And it made life easier.
```
Comment (whowutwut) [inline docs/source/guides/admin-guides/manage_clusters/ppc64le/management.rst]: suggest: 

Most of modern enterprise level server machine**s** ~~has an~~ have LED**s** installed on its front panel and/or rear panel**,** which are ~~. Which is~~ called beacon light**s**. **When** ~~While~~ this light has been turned on, the system ~~adminstrator~~**administrator** can use this light to indicate one ~~physcial~~ **physical** machine out of a bunch of enclosures in a server frame. ~~And it made life easier.~~ It makes life easier!

Note: administrator and physical are spelled wrong

File: docs/source/guides/admin-guides/manage_clusters/ppc64le/management.rst [hunk 3]
```diff
@@ -0,0 +1,113 @@
...
+
+Please notice, the current state of the beacon light can not be inquery remotely. As a workaround, one can always use the `rbeacon` command to trun all the beacon lights in one frame off, and then turn a particular beancon light on. ::
+
+    rbeacon f1cn off
+    rbeacon cn5 on
+
+Remote Power Control
+````````````````````
+
+The next important thing is to control the power of a remote physical machine. For this propose, `rpower` command is involved. ::
```
Comment (whowutwut) [inline docs/source/guides/admin-guides/manage_clusters/ppc64le/management.rst]: propose -> purpose 
same comment on the commands, should we add a extra back tick to make it code format? ``rpower``

File: docs/source/guides/admin-guides/manage_clusters/ppc64le/management.rst [hunk 4]
```diff
@@ -0,0 +1,113 @@
...
+
+Get the current rpower state of a machine, please refer the example below. ::
+
+    # rpower cn1 state
+    cn1: Running
+
+Remote Console
+``````````````
+
+Nowaday, most enterprise level servers do not have video adapter installed along with the machine. That means, the end user can not connect a regular monitor to the machine and get display output. In most case, the console can be get from serial port, or LAN port, through Serial-over-LAN. Serial cable or network cable is used to get a command line interface of the machine. From there, the end user can get the basic machine booting information, firmware settings interface, local command line console, and something like that.
```
Comment (whowutwut) [inline docs/source/guides/admin-guides/manage_clusters/ppc64le/management.rst]: Suggest the following changes:

~~Nowaday, m~~ **M**ost enterprise level servers do not have video adapter**s** installed ~~along~~ with the machine. **Meaning**~~That means~~, the end user can not connect a ~~regular~~ monitor to the machine and get display output. In most case**s**, the console can be **viewed using the** ~~get from~~ serial port~~,~~ or LAN port, through Serial-over-LAN. Serial cable or network cable **are** ~~is~~ used to get a command line interface of the machine. From there, the end user can get the basic machine booting information, firmware settings interface, local command line console, **etc**. ~~and something like that.~~

File: docs/source/guides/admin-guides/manage_clusters/ppc64le/management.rst [hunk 5]
```diff
@@ -0,0 +1,113 @@
...
+
+In order to get the command line console remotely. xCAT introduce the `rcons` command. ::
+
+
+First of all, make sure the `conserver` is configured, if it is not, configue it with ::
```
Comment (whowutwut) [inline docs/source/guides/admin-guides/manage_clusters/ppc64le/management.rst]: Suggest: 
... , if not, configure with ::

File: docs/source/guides/admin-guides/manage_clusters/ppc64le/management.rst [hunk 6]
```diff
@@ -0,0 +1,113 @@
...
+
+    # rpower cn1 state
+    cn1: Running
+
+Remote Console
+``````````````
+
+Nowaday, most enterprise level servers do not have video adapter installed along with the machine. That means, the end user can not connect a regular monitor to the machine and get display output. In most case, the console can be get from serial port, or LAN port, through Serial-over-LAN. Serial cable or network cable is used to get a command line interface of the machine. From there, the end user can get the basic machine booting information, firmware settings interface, local command line console, and something like that.
+
+In order to get the command line console remotely. xCAT introduce the `rcons` command. ::
```
Comment (whowutwut) [inline docs/source/guides/admin-guides/manage_clusters/ppc64le/management.rst]: suggest:  xCAT provides the ``rcons`` command. ::

File: docs/source/guides/admin-guides/manage_clusters/ppc64le/management.rst [hunk 7]
```diff
@@ -0,0 +1,113 @@
...
+
+First of all, make sure the `conserver` is configured, if it is not, configue it with ::
+
+    makeconservercf
+
+Then check if the `conserver` is up and running ::
+
+    ps ax | grep conserver
+
+If the server is not running, or you just updating its configuration file, restart the server with ::
```
Comment (whowutwut) [inline docs/source/guides/admin-guides/manage_clusters/ppc64le/management.rst]: suggest "updating" --> "updated"

File: docs/source/guides/admin-guides/manage_clusters/ppc64le/management.rst [hunk 8]
```diff
@@ -0,0 +1,113 @@
...
+
+    rcons cn1
+
+Advanced operation
+------------------
+
+Remote Hardware Inventory
+`````````````````````````
+
+When you have a lot of physical machines in one place, the most important thing is identify which is which. Mapping the model type and/or serial number of a machine with its host name. Command `rinv` is involved in such a situation. Which this command, most of the important information to distinct one machine from all the others can be get, remotely.
```
Comment (whowutwut) [inline docs/source/guides/admin-guides/manage_clusters/ppc64le/management.rst]: suggest changing "get" at the end to "obtained remotely."

File: docs/source/guides/admin-guides/manage_clusters/ppc64le/management.rst [hunk 9]
```diff
@@ -0,0 +1,113 @@
...
+Most of modern enterprise level server machine has an LED installed on its front panel and/or rear panel. Which is called beacon light. While this light has been turned on, the system adminstrator can use this light to indicate one physcial machine out of a bunch of enclosures in a server frame. And it made life easier.
+
+With xCAT, the end user can turn the beacon light on or off with the commands show below. ::
+
+    rbeacon cn1 on
+    rbeacon cn1 off
+
+Please notice, the current state of the beacon light can not be inquery remotely. As a workaround, one can always use the `rbeacon` command to trun all the beacon lights in one frame off, and then turn a particular beancon light on. ::
+
+    rbeacon f1cn off
```
Comment (whowutwut) [inline docs/source/guides/admin-guides/manage_clusters/ppc64le/management.rst]: should be cn1 not f1cn?

File: docs/source/guides/admin-guides/manage_clusters/ppc64le/management.rst (hunk 1 above)
Comment (whowutwut) [inline docs/source/guides/admin-guides/manage_clusters/ppc64le/management.rst]: "beancon" spelled wrong

File: docs/source/guides/admin-guides/manage_clusters/ppc64le/management.rst [hunk 10]
```diff
@@ -0,0 +1,113 @@
...
+
+As an example, get only the temperature information of a particular machine. ::
+
+    rvitals cn1 temp
+
+Firmware Updating
+`````````````````
+**TODO**: For OpenPower machine, the firmware updating featue is not implement in the `rflash` command. The section should be updated after this feature get implemented.
+
+Configures Nodes' Service Processors
```
Comment (whowutwut) [inline docs/source/guides/admin-guides/manage_clusters/ppc64le/management.rst]: Can we just write "Configure Service Processors" ?

File: docs/source/guides/admin-guides/manage_clusters/ppc64le/management.rst [hunk 11]
```diff
@@ -0,0 +1,113 @@
...
+    rinv cn1 all
+
+As an example, in order to get only the information of firmware version, the follwing command can be used. ::
+
+    rinv cn1 firm
+
+Remote Hardware Vitals
+``````````````````````
+
+Collect runtime information from running physical machine is also such a big requirement for real life system administrators. This kind of information includes, temperature of CPU, internal voltage of paricular socket, wattage with workload, speed of cooling fan, et al.
```
Comment (whowutwut) [inline docs/source/guides/admin-guides/manage_clusters/ppc64le/management.rst]: suggest removing "such"   .. "is also a big requirement.."

File: docs/source/guides/admin-guides/manage_clusters/ppc64le/management.rst [hunk 12]
```diff
@@ -0,0 +1,113 @@
...
+
+    rvitals cn1 all
+
+As an example, get only the temperature information of a particular machine. ::
+
+    rvitals cn1 temp
+
+Firmware Updating
+`````````````````
+**TODO**: For OpenPower machine, the firmware updating featue is not implement in the `rflash` command. The section should be updated after this feature get implemented.
```
Comment (whowutwut) [inline docs/source/guides/admin-guides/manage_clusters/ppc64le/management.rst]: Suggest:   ... the firmware updating feature is not implemented in the ``rflash`` command.

**Comment by whowutwut** [comment]
Thanks for writing up this page, looks good!

**Comment by whowutwut** [comment]
Looks good. Let's merge it.

**Pull request merged**